platforms/core-runtime/base-services/src/main/java/org/gradle/internal/operations/BuildOperationExecutor.java

```diff
@@ -24,56 +24,11 @@
 import javax.annotation.concurrent.ThreadSafe;
 
 /**
- * Executes build operations synchronously, asynchronously or via a {@link BuildOperationQueue}.
- *
- * The executor provides several capabilities:
- *
- * <ul>
- *     <li>Fires events via {@link BuildOperationListener}. For example, this means that notification of build operation
- *     execution can be received by tooling API clients.</li>
- *     <li>Generates progress logging events.</li>
- * </ul>
+ * Executes build operations via a {@link BuildOperationQueue}.
  */
```

Comment:
💭 This change signifies the point of the whole effort here: we can now express what this class is about a lot simpler.